main.go

```diff
@@ -223,27 +225,26 @@ func findBrowser(target string) string {
 }
 
 // getBrowserNameByOS normilizes browser name
-func getBrowserNameByOS(k string) string {
-	browser := ""
-	switch k {
-	case "google", "chrome":
-		switch runtime.GOOS {
-		case "darwin":
-			browser = "Google Chrome"
-		}
-	case "mozilla", "firefox":
-		switch runtime.GOOS {
-		case "darwin":
-			browser = "Firefox"
-		}
-	case "brave":
-		switch runtime.GOOS {
-		case "darwin":
-			browser = "Brave"
+func getBrowserNameByOS(denormalizedStr string) string {
+	os := runtime.GOOS
+
+	// key = denormalized value, value = normalized value
+	browserNameMap := map[string]string{
+		"google":  "Google Chrome",
+		"chrome":  "Google Chrome",
+		"mozilla": "Firefox",
+		"firefox": "Firefox",
+		"brave":   "Brave",
+	}
+
+	if os == "darwin" {
```

Comment:
I was thinking if we can refactor this to support Windows and Linux.

Reply:
I add this issue. there is a list of all the OS name. the browser string may be vary between each OS.
https://github.com/Bunchhieng/hnreader/issues/21